src/app/combobox/combobox.component.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+
+export interface ComboboxItem {
+  value: string;
+  label: string;
+  lowercaseLabel: string;
+}
+
+/**
+ * Sample component
+ *
+ * Usage:
+ * <code><pre>
+ * // Individual module import
+ * import { SampleModule } from 'patternfly-ng/sample';
+ * // Or
+ * import { SampleModule } from 'patternfly-ng';
+ *
+ * &#64;NgModule({
+ *   imports: [SampleModule,...]
+ * })
+ * export class AppModule(){}
+ * </pre></code>
+ */
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'pfng-combobox',
+  styles: [`
+    .combobox-disabled .dropdown-toggle {
+      cursor: not-allowed;
+      background-image: none;
+      box-shadow: none !important;
+    }
+  `],
```

Comment:
Please create a stylesheet for the component and add that to patternfly-ng.less. We're not embedding CSS in the components.